src/helm/clients/openai_client.py

```diff
@@ -191,6 +187,18 @@ def _make_chat_request(self, request: Request) -> RequestResult:
             if raw_request["stop"] is None:
                 raw_request.pop("stop")
 
+            if request.model_engine == "o1-2024-12-17":
+                # Avoid error:
```

Comment:
@yifanmai note the extra logic for `o1-2024-12-17` that's different from the other o1 models.